src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.shared.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.ObjectModel;
+using System.Windows.Input;
+using Microsoft.Maui.Controls;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// DrawingView Platform Control
+/// </summary>
+public partial class MauiDrawingView
+{
+	readonly WeakEventManager weakEventManager = new();
+
+	/// <summary>
+	/// Event raised when drawing line completed 
+	/// </summary>
+	public event EventHandler<MauiDrawingLineCompletedEventArgs> DrawingLineCompleted
+	{
+		add => weakEventManager.AddEventHandler(value);
+		remove => weakEventManager.RemoveEventHandler(value);
```

Comment:
Dang - I just realized that my `Move WeakEventManager to Core` PR hasn't merged yet, meaning we can't use `WeakEventManager` in `CommunityToolkit.Maui.Core` yet 🤦‍♂️ https://github.com/dotnet/maui/pull/5139

Reply:
It is merged, and we can finally use it!